README.md

```diff
@@ -210,11 +210,32 @@ app.use(limitMiddleware.middleware(options, function(req, res, next) {
 }));
 ```
 
+Include ratelimit response headers:
+
+```javascript
+var options = {
+  headers: true
```

Comment:
I don't think that's a bad idea at all. Given the current implementation, a user is free to define his or her own middleware and operate on `req.ratelimitState` as well, including whatever headers are desired. With that being said, more flexibility isn't a bad idea.